royaloperahouse/stream-tv-app
# Hotfix/improving of receiving video data

#60 Prismic investigation how data is being pulled through

**What?**

Fix problem with passing an empty string as URL of Akamai Manifest to Bitmovin Player

**Why?**

for seeing a video on Bitmovin player

**How?**

 moving to fetch video data from Redux saga`s loop to press button event (When user want to do it)

**How to test?**

When we press on the button "watch now"  we will see a video in Bitmovin player(if the video exists) in subscribed mode
